bakerydemo/settings/base.py

```diff
@@ -149,21 +149,27 @@
 
 
 # Internationalization
-# https://docs.djangoproject.com/en/3.2/topics/i18n/
+# https://docs.djangoproject.com/en/stable/topics/i18n/
 
 LANGUAGE_CODE = "en-us"
 
 TIME_ZONE = "UTC"
 
 USE_I18N = True
 
-USE_L10N = True
-
-USE_TZ = True
```

Comment:
Even though `USE_TZ` defaults to `True` in Django 5.0+, it's still included in the default template: https://github.com/django/django/blob/240421c7c4c81fe5df26274b807266bd4ca73d7f/django/conf/project_template/project_name/settings.py-tpl#L111

So I think we can keep this in.